app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class PasswordResetsController < ApplicationController
+  before_action :get_user,         only: %i[edit update]
+  before_action :valid_user,       only: %i[edit update]
+  before_action :check_expiration, only: %i[edit update]
+
+  def new; end
+
+  def create
+    @user = User.find_by(email: params[:password_reset][:email].downcase)
+    
+    if @user
+      @user.create_reset_digest
+      @user.send_password_reset_email
+      flash[:info] = 'Email sent with password reset instructions'
+      redirect_to login_path
```

Comment:
newline above `redirect_to login_path` makes it easier to read. I usually separate the return in a method as well if it's the last line, or the final action. You can use new lines to separate separate actions/code blocks